packages/cli/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "scripts": {
     "build": "tsc --build",
-    "lint": "tslint -p .",
+    "lint": "tsc --composite false --noEmit && eslint src --ext .js,.ts,.tsx",
```

Comment:
Miksi koodi pitäisi erikseen tyyppitarkistaa ennen linttiä ?

Reply:
No toi on aika yleinen konventio että tarkistetaan tyypit samalla kun tarkistetaan lintteri virheet. Saa nuokin yksinkertaiset virheet veke nopeasti samalla kun saadaan linterin löytämät virheet. Ei tietty ole pakollista. En ole varma kumminpäin on kivempi

Reply:
Se on turhaa työtä.

CI-jobi `yarn && yarn test -i --ci && yarn lint` kääntää koodin jo heti alussa, sitä ei tarvitse tehdä enää uudestaan lint-vaiheessa. 

Muutenkin oletus on, että devaajilla on koko ajan `yarn watch` päällä, joten koodi käännetään koko ajan reaaliaikaisesti.

Reply:
👍 Mä otan tsc:n veke linttauksesta. Katsotaan jos sen puuttumisesta tulee oikeasti ongelmia.